src/components/__tests__/FontAwesomeIcon.test.js

```diff
@@ -33,7 +37,7 @@ if(coreHasFeature(ICON_ALIASES)) {
     const vm = mountFromProps({ icon: ['fas', 'close'] })
 
     expect(vm.$el.tagName).toBe('svg')
-    expect(vm.$el.classList.contains('fa-close')).toBeTruthy()
+    expect(vm.$el.classList.contains('fa-xmark')).toBeTruthy()
```

Comment:
did you want the `fa-xmark` icon here?  
In v6 `xmark` is the name of the icon and `close` is the alias.
If we keep the `fa-mark` we will need to import it (line 4), and will probably want to change the test name; currently test is 'find a free-solid-svg-icon that is an alias'.
As of right now, the test mentioned on the previous line fails because `xmark` has NOT been imported.